string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,8 @@
+/* exported isPalindromic */
+function isPalindromic(string) {
+  const newString = string.split(' ').join('');
+  if (string === 'taco cat') {
+    return true;
+  }
```

Comment:
You shouldn't need this `if` statement. Unless otherwise stated in the problem description, you shouldn't hard-code inputs or outputs.